### ISSUE TITLE
Cannot set Date formatting by 10 minute steps

### ISSUE THREAD

**Issue by cdaniel77** [IonDen/ion.rangeSlider#364]
I have included the fiddle here:  http://jsfiddle.net/qv6yrjrv/425/

I am trying to show just minutes in steps of 10.  But do not understand how to get the slider to move in minutes of time.

**Comment by IonDen** [comment]
Hi, please study this demo: http://jsfiddle.net/IonDen/w1ucb8bm/
As you can see, slider does not work with dates itself, it works with its int value. So, every value you set in to slider, at first you should covert to number, like this: `+new Date()`, the same for step. You should convert you 10 mins step to miliseconds/seconds.

**Comment by cdaniel77** [comment]
Thank you that solved my issue.  One last question, is there a way to reset the slider?  Let's say I updated the max and min, is there a way to update the slider to reflect the new values?

**Comment by IonDen** [comment]
[Of course you can](https://github.com/IonDen/ion.rangeSlider#public-methods)

**Comment by cdaniel77** [comment]
Okay, I found my issue.  There is a slight "enhancement" needed for the update.  If you have conditions in the prettify portion of the slider the conditions are not fired on update.  For example, if you have an if statement that changes the end points and fire an update the prettify will not reevaluate the conditions.  it will just keep the initial condition that was loaded.  To fix I just had to pass in the prettify option to the update.  It would be nice to not have to do that and the prettify portion is called again on update.
